### PR TITLE
Snow projection discovery

### PR DESCRIPTION
**Pull request by jupflug**
Creating a data discovery for the climate and snow projection datasets

*** Please don't merge this pull request to the main branch yet, issuing this pull request for preview purposes internally ***

### CODE REVIEW

File: discoveries/projected-changes-WUS-snow.discoveries.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    <Caption
+      attrAuthor='NASA'
+      attrUrl='https://nasa.gov/'
+    >
+    Median projected percent-changes to end-of-century total winter precipitation from a 23-member ensemble of CMIP6 models (SSP2-4.5).
+</Caption>  
+  </Figure>
+</Block>
+
+<Block type='wide'>
```
Comment (danielfdsilva) [inline discoveries/projected-changes-WUS-snow.discoveries.mdx:73]: Consider whether to keep the `wide` block for such little text, as it shows up in a partial column looking broken.

image

File: discoveries/projected-changes-WUS-snow.discoveries.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+       alt="changes to thresholded habitat across elevation bands" 
+      />
+   </Figure>
+   <Prose>
+   The snow projections developed here can be used for multiple applications. For example, wolverine habitat assessments performed by the US Fish and Wildlife Service suggested that snowy regions were more likely to support denning if mid-spring (e.g., May 15) snow depth exceeded 0.5m. Using these projections, we could identify the percent-change to the amount of area that had snow depth exceeding the spring snow depth threshold for different elevation bands. The plots shown here show the changes across elevation bands for the four future periods (emissions scenario SSP2-4.5), for three domains, accounting for both the ensemble mean (line) and spread (shaded) across 23 different climate models. Note that although some of the colder domains like the Colorado Rockies and Montana Rockies had high-elevation increases to end-of-century projected snow, the prevelence of area exceeding this habitat threshold only decreased for all elevation bands in the last simulation period (2076 - 2095).
+   </Prose>
+</Block>  
+
+<Block type='wide'>
```
Comment (danielfdsilva) [inline discoveries/projected-changes-WUS-snow.discoveries.mdx:138]: Consider whether to keep the `wide` block for such little text, as it shows up in a partial column looking broken.

**Review by danielfdsilva — CHANGES_REQUESTED** [review]
All the collections used here are missing the `dashboard:time_density` property, which leads to problems. _This is a data situation, not about this configuration_

- https://staging-stac.delta-backend.com/collections/CMIP245-winter-median-pr
- https://staging-stac.delta-backend.com/collections/CMIP585-winter-median-pr
- https://staging-stac.delta-backend.com/collections/CMIP245-winter-median-ta
- https://staging-stac.delta-backend.com/collections/CMIP585-winter-median-ta

cc @anayeaye @smohiudd 

---

@jupflug
- The images are a bit to heavy and should be cropped/compressed. You cant find [guidance in our docs](https://github.com/NASA-IMPACT/veda-config/blob/cf883f64238a63871f76c614faa4063ec74e3b9f/docs/frontmatter/media.md).
- There are several csv files included in this PR that don't seem to be used. Consider removing them if not needed.

**Comment by danielfdsilva** [comment]
@ividito This sort of works but so far for yearly datasets the date was set at the beginning of the year (jan 1st) since on time step represent the whole year. In this case it is set to feb 14th, and it breaks things a bit.
Is this something to be expected: yearly datasets that fall on specific days? Can the dates be changed?

File: discoveries/projected-changes-WUS-snow.discoveries.mdx (hunk 2 above)
Comment (danielfdsilva) [reply discoveries/projected-changes-WUS-snow.discoveries.mdx:138]: @jupflug Should this block be wide (last in the page)

image

**Comment by jupflug** [comment]
Sorry @danielfdsilva. I'm not sure how I missed the two of those. The newest push should have the reduced jpgs (both around 300 KB).

**Pull request merged**